.github/workflows/ci.yaml

```diff
@@ -0,0 +1,128 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/**'
+      - proton.json
+      - truckersmp-cli
+      - truckersmp-cli.c
+  pull_request:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/**'
+      - proton.json
+      - truckersmp-cli
+      - truckersmp-cli.c
+
+env:
+  OUTDIR: /tmp/output.d
```

Comment:
* ~~It's possible to be used in multiple jobs in the future.~~
* ~~Located near the top of the file.~~

Since the file has been split it's outdated, though.